### DISCUSSION TITLE
How to cross compile lapack?

### DISCUSSION THREAD

**Discussion by atptour2017** [Reference-LAPACK/lapack#842] [Q&A]
I want to cross compile lapack for armv8 CPU.
I have define the fortran complier (aarch64-kedacom-linux-gnu-gfortran) but it still check complier by itself:

(base) [root@localhost build]# cmake -DCMAKE_C_COMPLIER=aarch64-kedacom-linux-gnu-gcc -DCMAKE_CXX_COMPLIER=aarch64-kedacom-linux-gnu-g++ -DCMAKE_FORTRAN_COMPLIER=aarch64-kedacom-linux-gnu-gfortran -DCMAKE_INSTALL_LIBDIR=/root/build/third_party_scipy/lapack/dist ..
-- The Fortran compiler identification is unknown
/usr/bin/f95.br_real: No such file or directory
-- The C compiler identification is unknown
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - failed
-- Check for working Fortran compiler: /usr/bin/f95
-- Check for working Fortran compiler: /usr/bin/f95 - broken

**Comment by martin-frbg** [comment]
CMAKE uses mixed case in the Fortran compiler option, try setting `-DCMAKE_Fortran_COMPILER=`...
(and if the above was a cut-and-paste of the actual terminal content, you have systematically mis-spelled  COMPILER as COMPLIER everywhere)

**Comment by weslleyspereira** [comment]
Hi @atptour2017. Was your issue fixed? Thanks
